src/vivarium_public_health/population/add_new_birth_cohorts.py

```diff
@@ -157,7 +157,8 @@ def setup(self, builder):
             Framework coordination object.
         """
         age_specific_fertility_rate = self.load_age_specific_fertility_rate_data(builder)
-        fertility_rate = builder.lookup.build_table(age_specific_fertility_rate, key_columns=())
+        fertility_rate = builder.lookup.build_table(age_specific_fertility_rate, key_columns=(),
```

Comment:
Do you need key cols, if so, provide a list instead of a tuple for consistency.

Reply:
made a small change in vivarium so that key cols can be None (the default if not provided) so I dropped these